cmd/operator-sdk/migrate/cmd.go

```diff
@@ -23,20 +23,28 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/ansible"
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/helm"
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/project"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
 
+	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
+var depManager string
+
 // NewCmd returns a command that will add source code to an existing non-go operator
 func NewCmd() *cobra.Command {
-	return &cobra.Command{
+	newCmd := &cobra.Command{
 		Use:   "migrate",
 		Short: "Adds source code to an operator",
 		Long:  `operator-sdk migrate adds a main.go source file and any associated source files for an operator that is not of the "go" type.`,
 		RunE:  migrateRun,
 	}
+
+	newCmd.Flags().StringVar(&depManager, "dep-manager", "dep", "Dependency manager the new project will use (choices: \"dep\")")
```

Comment:
Just to highlight that we're adding a new flag here. Normally this would warrant a mention in the CHANGELOG but I think we'll probably have to mention that once we have the actual implementation and docs for using go mod.

However, can you still add this flag to the CLI reference guide here just so we can update it #1001 or later with the go mod option.